monai/apps/reconstruction/mri_utils.py

```diff
@@ -0,0 +1,202 @@
+        track_meta: whether to track the meta information, if `True`, will convert to `MetaTensor`.
+            default to `False`.
+
+    Returns:
+        PyTorch version of the data
+
+    Example:
+        .. code-block:: python
+
+            import numpy as np
+            data = np.array([ [1+1j, 1-1j], [2+2j, 2-2j] ])
+            # the following line prints (2,2)
+            print(data.shape)
+            # the following line prints torch.Size([2, 2, 2])
+            print(convert_to_tensor_complex(data).shape)
+    """
+    # if data is not complex, just turn it into a tensor
+    if isinstance(data, Tensor):
+        if not torch.is_complex(data):
+            converted_data: Tensor = convert_to_tensor(
+                data, dtype=dtype, device=device, wrap_sequence=wrap_sequence, track_meta=track_meta
+            )
+            return converted_data
+    else:
+        if not np.iscomplexobj(data):
+            converted_data = convert_to_tensor(
+                data, dtype=dtype, device=device, wrap_sequence=wrap_sequence, track_meta=track_meta
+            )
+            return converted_data
+
+    # if data is complex, turn its stacked version into a tensor
+    if isinstance(data, torch.Tensor):
+        data = torch.stack([data.real, data.imag], dim=-1)
+
+    elif isinstance(data, np.ndarray):
+        if re.search(r"[SaUO]", data.dtype.str) is None:
+            # numpy array with 0 dims is also sequence iterable,
+            # `ascontiguousarray` will add 1 dim if img has no dim, so we only apply on data with dims
+            if data.ndim > 0:
+                data = np.ascontiguousarray(data)
+            data = np.stack((data.real, data.imag), axis=-1)
+
+    elif isinstance(data, (float, int)):
+        data = [[data.real, data.imag]]
+
+    elif isinstance(data, list):
+        data = convert_to_numpy(data, wrap_sequence=True)
+        data = np.stack((data.real, data.imag), axis=-1).tolist()
+
+    converted_data = convert_to_tensor(
+        data, dtype=dtype, device=device, wrap_sequence=wrap_sequence, track_meta=track_meta
+    )
+    return converted_data
+
+
+def complex_abs(x: NdarrayOrTensor) -> NdarrayOrTensor:
+    """
+    Compute the absolute value of a complex array.
+
+    Args:
+        x: Input array/tensor with 2 channels in the last dimension representing real and imaginary parts.
+
+    Returns:
+        Absolute value along the last dimention
+
+    Example:
+        .. code-block:: python
+
+            import numpy as np
+            x = np.array([3,4])[np.newaxis]
+            # the following line prints 5
+            print(complex_abs(x))
+    """
+    if x.shape[-1] != 2:
+        raise ValueError(f"x.shape[-1] is not 2 ({x.shape[-1]}).")
+    return (x[..., 0] ** 2 + x[..., 1] ** 2) ** 0.5
+
+
+def root_sum_of_squares(x: NdarrayOrTensor, spatial_dim: int) -> NdarrayOrTensor:
+    """
+    Compute the root sum of squares of the data (typically done for multi-coil MRI samples)
```

Comment:
Compute the root sum of squares of the data --> Compute the root sum of squares (rss) of the data